components/gene-panels/ui/src/main/resources/PhenoTips/PanelsDisplay.xml

```diff
@@ -400,7 +400,10 @@
       this.baseUrl = service.script + "/livetable";
       this.saveButton = saveButton;
       this.checkBox = $('excludeRejectedGenesCheckBox');
-      this.checkBox &amp;&amp; (this.excludeRejected = this.checkBox.checked);
+      if (this.checkBox) {
+        this.excludeRejected = true;
+        this.checkBox.checked = this.excludeRejected;
+      }
```

Comment:
OR `this.checkBox &amp;&amp; (this.excludeRejected = this.checkBox.checked = true);`